tokens/manage/multisig-safe.md

```diff
@@ -312,4 +313,36 @@ To double check that the correct text was set, you can go through the process ag
 
 Congratulations, you've successfully interacted with a smart contract using the Safe!
 
+## Using Moonbeam Safe APIs
+There are APIs available to read from and interact with Moonbeam Safes for Moonbase Alpha, Moonriver, and Moonbeam.
+
+|     Network    |                                           Multisig API Page                                              |
+|:--------------:|:--------------------------------------------------------------------------------------------------------:|
+|    Moonbeam    |  [{{ networks.moonbeam.multisig.api_page }}]({{ networks.moonbeam.multisig.api_page }}){target=_blank}   |
+|    Moonriver   | [{{ networks.moonriver.multisig.api_page }}]({{ networks.moonriver.multisig.api_page }}){target=_blank}  |
+| Moonbase Alpha |  [{{ networks.moonbase.multisig.api_page }}]({{ networks.moonbase.multisig.api_page }}){target=_blank}   |
+
+### Retrieve Safe Information
```

Comment:
```suggestion
### Retrieve Safe Information {: #retrieve-safe-information }

```

Reply:
actually I don't know if this header is even necessary. I would probably just remove it and mention you can get all sorts of information about the safe, transactions, and more and this example will just show you how to use the API to get information about the safe

Reply:
I would say that we should point at this being an example, but that there are many other methods available that you can check in each of the URLs?

Reply:
Will remove the header & point to the fact that it's an example